src/app.ts

```diff
@@ -1,8 +1,11 @@
 import express from 'express';
+import expressWinston from 'express-winston';
 import bodyParser from 'body-parser';
 import { apiRoutes } from './routes';
+import { logger } from './log';
 
 const app = express();
+app.use(expressWinston.logger({ level: 'debug', winstonInstance: logger }));
```

Comment:
After looking at [the log levels](https://www.npmjs.com/package/winston#logging-levels) more carefully, I should probably be using level `http` here; I will fix tomorrow.